src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -345,15 +389,14 @@ export default class OrgPerson extends Mixins(CommonMixin) {
   /** Model value for roles checkboxes. */
   private selectedRoles: Array<RoleTypes> = []
 
-  /** Model for Organization look up. */
-  private toggleBcLookUp = true
```

Comment:
This is now a property in the orgPerson object (so that we know, in the future, whether the org-person was looked up or entered manually).